editor/components/arrangement-toolbar/index.js

```diff
@@ -0,0 +1,52 @@
+/**
```

Comment:
@jasmussen I think we need to take a look at the glossary and figure out how to describe and group things. I've been thinking we could do `block-controls` and within that have the different pieces: toolbar, placement, switcher, etc.